frontend/src/scenes/pipeline/PipelineNodeConfiguration.tsx

```diff
@@ -0,0 +1,133 @@
+import { LemonButton, LemonSkeleton, Tooltip } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { IconLock } from 'lib/lemon-ui/icons'
+import { LemonMarkdown } from 'lib/lemon-ui/LemonMarkdown'
+import React from 'react'
+import { BatchExportsEditFields } from 'scenes/batch_exports/BatchExportEditForm'
+import { BatchExportConfigurationForm } from 'scenes/batch_exports/batchExportEditLogic'
+import { getConfigSchemaArray, isValidField } from 'scenes/pipeline/configUtils'
+import { PluginField } from 'scenes/plugins/edit/PluginField'
```

Comment:
could we move the code to this side & import from the other side, would be easier to nuke later

Reply:
We can, but it make this PR even less readable, so I think it's worth doing that in a separate PR